packages/fuel-gauge/src/contract.test.ts

```diff
@@ -754,18 +806,15 @@ describe('Contract', () => {
     expect(result.status).toBe('success');
   });
 
-  it('Provide a custom provider and public wallet to the contract instance', async () => {
-    const contract = await setupContract();
+  // #TODO: Discuss whether this test is still relevant
+  it.skip('Provide a custom provider and public wallet to the contract instance', async () => {
```

Comment:
In my opinion this test is not relevant as anyone who wants to provide a custom provider should have written their tests for a custom provider, this test just seeks emulate one instance of such custom provider

Reply:
Removed in 4e27d7723